tasks/Tasks/provision_self_managed_cluster.robot

```diff
@@ -36,3 +37,9 @@ Delete GPU Node From Self-Managed AWS Cluster
     [Tags]    gpu_node_aws_self_managed_deprovision
     Login To Cluster
     Delete GPU Node In Self Managed AWS Cluster
+
+Disconnect Self-Managed Cluster
+    [Documentation]    Disconnect a self-managed cluster
+    [Tags]    self_managed_disconnect
+    Login To Cluster
```

Comment:
is this `Login To Cluster` refers to loggin into Hive right? If so, could you make the kw explicitly mentioning that?

Reply:
Deleting Login To Cluster to allow the ability to choose between different login options and avoid conflicts

Reply:
so, I assume this refers to the Hive cluster right?